src/graph/ShowExecutor.cpp

```diff
@@ -53,6 +51,12 @@ void ShowExecutor::execute() {
         case ShowSentence::ShowType::kShowEdges:
             showEdges();
             break;
+        case ShowSentence::ShowType::kShowTagIndexes:
+            showTagIndexes();
+            break;
+        case ShowSentence::ShowType::kShowEdgeIndexes:
+            showEdgeIndexes();
+            break;
```

Comment:
could we provide a method for show index? for example : 
show tag index(name)
indexID | indexName | col1 | col2 | col3 | ....

Reply:
Aha , showCreateTagIndex and showCreateEdgeIndex seems to have the same purpose